org.jacoco.core.test.validation.kotlin/src/org/jacoco/core/test/validation/kotlin/targets/KotlinGeneratedTarget.kt

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2018 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Fabian Mastenbroek - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.test.validation.kotlin.targets
+
+import org.junit.Assert.assertEquals
+
+/**
+ * Test target for Kotlin generated methods
+ */
+object KotlinGeneratedTarget {
+    private fun testTrivial() {
+        data class Target(var value: Int)  // assertFullyCovered()
+
+        val target = Target(0)
+        target.value = 1
+        assertEquals(1, target.value)
```

Comment:
Assertions in the test targets itself look a bit confusing to me. To trigger execution we're typically using our `Stubs`, e.g. in this case:

    nop(target.value)

Reply:
👍 Thx for quick update!